### PR TITLE
fix(Invite & support): command with Discord embed

### PR DESCRIPTION
**Pull request by deadshotofficial**
This PR is same as the previous one just that I will do it bit-by-bit starting off with Utilities Section

### CODE REVIEW

**Review by officialpiyush — CHANGES_REQUESTED** [review]
I think it would be better if we check if the bot has `EMBED_LINKS` permission in the channels

**Comment by XynoxTheDev** [comment]
> I think it would be better if we check if the bot has `EMBED_LINKS` permission in the channels

Yes, you are right @officialpiyush but @deadshotofficial is there really a need to have embedded for such commands providing very little information?

**Comment by deadshotofficial** [comment]
@XynoxTheDev it is for adding to looks of commands, making it more attracting. and yeah about the PR Template, i just learned about it. for the next commands, i will check that too

**Comment by XynoxTheDev** [comment]
> @XynoxTheDev it is for adding to looks of commands, making it more attracting. and yeah about the PR Template, i just learned about it. for the next commands, i will check that too

I think these commands are fine and are simple to be used without any permission checks. You can contribute by adding new features and upgrading other commands which are used very often not as utility commands.

**Comment by Anish-Shobith** [comment]
well its quite easy @deadshotofficial ,
```js
const { Command } = require('discord-akario');

class ExampleCommand extends Command {

	constructor() {
		super('example', {
			clientPermission: 'EMBED_LINKS'
		})
	}
}
```

File: src/commands/Utilities/invite.js [hunk 1]
```diff
@@ -13,10 +14,17 @@ class Invite extends Command {
     }
 
     exec(message) {
+        const embed = MessageEmbed()
+        .setColor('#00ff9e')
+        .setTitle('Want To Invite Me?')
+        .addField('Invite Link ❯', [
+            '[Add Me Now!](https://discord.com/api/oauth2/authorize?client_id=759763855680602122&permissions=8&redirect_uri=https%3A%2F%2Fdiscord.com%2Finvite%2FZzbZpdw&scope=bot)'
+        ])
+        .setFooter(`${this.client.user.username} is made with ❤️`, `https://cdn.discordapp.com/emojis/805614116937007165.png?v=1`)
```
Comment (Anish-Shobith) [inline src/commands/Utilities/invite.js:23]: semi colon at the end please

File: src/commands/Utilities/invite.js [hunk 2]
```diff
@@ -13,10 +14,17 @@ class Invite extends Command {
...
 
-        message.reply('**Invite me ❯ ** \nhttps://discord.com/api/oauth2/authorize?client_id=759763855680602122&permissions=8&redirect_uri=https%3A%2F%2Fdiscord.com%2Finvite%2FZzbZpdw&scope=bot')
+        message.channel.send(embed)
```
Comment (Anish-Shobith) [inline src/commands/Utilities/invite.js:25]: semi colon at the end please.

File: src/commands/Utilities/support.js [hunk 3]
```diff
@@ -13,10 +14,15 @@ class Support extends Command {
     }
 
     exec(message) {
+        const embed = new MessageEmbed()
+        .setColor('#00ff9e')
+        .setTitle('Want Some Support?')
+        .setDescription('Join ❯ https://discord.gg/ZzbZpdw')
+        .setFooter(`${this.client.user.username} is made with ❤️`, `https://cdn.discordapp.com/emojis/805614116937007165.png?v=1`)
```
Comment (Anish-Shobith) [inline src/commands/Utilities/support.js:21]: semi colon at the end please.

File: src/commands/Utilities/support.js [hunk 4]
```diff
@@ -13,10 +14,15 @@ class Support extends Command {
...
 
-        message.reply('**Join ❯ ** \nhttps://discord.gg/ZzbZpdw')
+        message.channel.send(embed)
```
Comment (Anish-Shobith) [inline src/commands/Utilities/support.js:23]: semi colon at the end please.

**Review by Anish-Shobith — CHANGES_REQUESTED** [review]

**Comment by XynoxTheDev** [comment]
Well, these changes are alright but I would make this PR a draft and it can be implemented in the future will further modification.

**Review by XynoxTheDev — COMMENTED** [review]
Well, these changes are alright but I would make this PR a draft and it can be implemented in the future will further modification.

**Comment by Anish-Shobith** [comment]
@deadshotofficial any progress on this ?

**Comment by XynoxTheDev** [comment]
This PR is inactive for so long and no response from the contributor is there till now, so I'm closing this issue.

**Pull request closed without merging**